course-template-bundle-spec.md

```diff
@@ -179,6 +181,46 @@ attribute        | required | type        | notes
 id               | ✓        | string      | A unique identifier for this step
 activity_options | ✓        | array       | `activity_options` is an array of dictionaries with the format:
 -- type          | ✓        | enum        | One of `lab`, `quiz`, `resource`
+-- subtype       |          | string      | A subordinate activity type; allowed `subtype` strings are contingent on the above `type`
```

Comment:
i was imagining that the author could use any string for this and that it would not be used outside of retake context. i don't think retake_policy_id makes sense though since i would like them to use some kind of subtype-like descriptor.

Reply:
Agreed.

1. I'm cool with it being an arbitrary string instead of enum, let's talk replace the 'allowed `subtype` strings' comment to reflect that.

2. I'm hesitant to call it `subtype`. It's not really a "type", but a use context.

How do y'all feel about:

- `usage`
- `usage_type`
- `use_case`
- `style`?
- `kind`?
- `disposition`?
- ...?

Reply:
sgtm. i don't know what the right word is. yours seem fine. here's more:
`context`?
`category`?

Reply:
`category` resonates most with me... Will send out a PR with that change / corresponding description update.  I presume it makes sense to update `activity_subtype` below (in `Retake Policies`) to `activity_category` as well, so will also include that change.